install.js

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+if (process.env['PUPPETEER_SKIP_CHROMIUM_DOWNLOAD']) {
+  console.log('**WARNING** Puppeteer is skipping chromium download because environment contains "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" variable.');
```

Comment:
Chromium

Reply:
Not sure this should say "Warning". Users opt-in to it. How about:

`console.log('**INFO** Skipping Chromium download. "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" environment variable was found.');`

Reply:
Done.